home-data-migration/src/main/resources/migration.properties

```diff
@@ -0,0 +1,4 @@
+url = jdbc:postgresql://localhost:5432/test
+changelog.path = home-data-migration\\src\\main\\resources\\db\\changelog
```

Comment:
I think this path isn't relevant for using this app outside of the project sources. Please use the one from classpath.